src/content/en/2020/jamstack.md

```diff
@@ -3,21 +3,265 @@
 part_number: III
 chapter_number: 17
 title: Jamstack
-description: Jamstack chapter of the 2020 Web Almanac covering the use of JamStack across the web including static site generators, static site hosting and headless CMSss
+description: Jamstack chapter of the 2020 Web Almanac covering the use of Jamstack, the performance of popular Jamstack frameworks, as well as an analysis of real-user experience using the Core Web Vitals metadata.
 authors: [ahmadawais]
-reviewers: [tpiros, phacks, YuLingCheng]
 analysts: [denar90, remotesynth]
+reviewers: [maedahbatool, phacks]
```

Comment:
Order is reviewers and then analsyts:

```suggestion
reviewers: [MaedahBatool, phacks]
analysts: [denar90, remotesynth]
```

@MaedahBatool / @remotesynth we didn;t have a profile for you so added the following in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1632/commits/95c230bd8579e512967111d55e3a1e6a22971453:

```

    "remotesynth": {
      "name": "Brian Rinaldi",
      "teams": [
        "analysts"
      ],
      "avatar_url": "https://avatars1.githubusercontent.com/u/216712?v=4&s=200",
      "website": "https://remotesynthesis.com/",
      "github": "remotesynth",
      "twitter": "remotesynth"
    },
```

```
    "MaedahBatool": {
      "name": "Maedah Batool",
      "teams": [
        "reviewers"
      ],
      "avatar_url": "https://avatars1.githubusercontent.com/u/12712988?v=4&s=200",
      "website": "https://maedahbatool.com/",
      "github": "MaedahBatool",
      "twitter": "maedahbatool"
    },
```

Let us know if you want any of that changed?